ansible/group_vars/all

```diff
@@ -5,4 +5,5 @@ histomicstk_version: master
 large_image_version: master
 dsa_version: master
 girder_worker_version: master
+itk_version: master
```

Comment:
Should we peg this to some latest stable release of ITK? I presume the master keeps changing rapidly.